maker-frontend/src/MakerApp.tsx

```diff
@@ -98,7 +98,20 @@ export default function App() {
     );
 
     const walletInfo = useLatestEvent<WalletInfo>(source, "wallet");
-    const priceInfo = useLatestEvent<PriceInfo>(source, "btcusd_quote");
+    const btcPriceInfo = useLatestEvent<PriceInfo>(source, "btcusd_quote");
+    const ethPriceInfo = useLatestEvent<PriceInfo>(source, "ethusd_quote");
+
+    let [priceInfo, setPriceInfo] = useState<PriceInfo | null>(null);
+
+    useEffect(() => {
+        if (symbol === Symbol.btcusd) {
+            setPriceInfo(btcPriceInfo);
+        } else if (symbol === Symbol.ethusd) {
+            setPriceInfo(ethPriceInfo);
+        } else {
+            setPriceInfo(null);
+        }
+    }, [btcPriceInfo, ethPriceInfo, symbol]);
```

Comment:
How often does this get executed? 
Given we have 1 event for `ethusd` and 1 event for  1 for `btcusd` I'm afraid that this gets fired too often.  

Again: we are the only one using this UI so I don't think this is a blocker but this is not ideal as this will slow down the frontend.

Reply:
in theory, it should be executed on quote subscription interval + when user changes the active contract symbol.

as there are 2 feeds updated on the same time (maybe it would be actually better to send the hashmap in the feed?), it fires 2x in short succession.

at times it seems to go a bit more often (I've seen new log line after 10s sometimes), but it's not too often IMHO.

Reply:
> as there are 2 feeds updated on the same time (maybe it would be actually better to send the hashmap in the feed?), it fires 2x in short succession.

yes, that's what I was thinking.